src/keyword-mark.ts

```diff
@@ -102,14 +117,15 @@ export class KeywordMarkElement extends HTMLElement {
 
     const text = this.textContent || '';
     const keywords = this.getAttribute('keywords');
+    const delimiter = this.getAttribute('delimiter') ?? /\s+/;
```

Comment:
and this one would then be `this.delimiter || /\s+/` (since delimiter will default to empty string)